BrowserKit/Sources/TabDataStore/TabDataStore.swift

```diff
@@ -22,10 +22,16 @@ public actor DefaultTabDataStore: TabDataStore {
     static let profilePath = "profile.profile"
     static let backupPath = "profile.backup"
     private var logger: Logger = DefaultLogger.shared
+    private var windowDataToSave: WindowData?
+    private var nextSaveIsScheduled = false
+    private let throttleTime: UInt64
 
-    public init() {}
+    public init(throttleTime: UInt64 = 5_000_000_000) {
```

Comment:
Q. Is that a special number or something random we picked? (5000,000,000)

Reply:
Pretty arbitrary, it's 5 seconds in nanoseconds. The API for using seconds with sleep is only available from iOS 16 up. I haven't settled on this being the final number, I think I'll chat with Daniela to figure it out. 5 seconds is the most often I would save but we could probably get away with maybe bumping it up to 15 seconds since we now have a back up mechanism too.

Reply:
Random thing: There's a global variable [NSEC_PER_SEC](https://developer.apple.com/documentation/dispatch/nsec_per_sec) which I usually use to better explain what is happening. In this case, it would be something like `5 * NSEC_PER_SEC`.